<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>471</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>276</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>276</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="lef_panel" native="true">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40,40,40);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="favourites_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/star.png</normaloff>
          <activeon>:/icons/icons/star_fav.png</activeon>:/icons/icons/star.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="database_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/database.png</normaloff>
          <activeon>:/icons/icons/database_fav.png</activeon>:/icons/icons/database.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="person_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/person.png</normaloff>
          <activeon>:/icons/icons/person_fav.png</activeon>:/icons/icons/person.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="event_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/event.png</normaloff>
          <activeon>:/icons/icons/event_fav.png</activeon>:/icons/icons/event.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="find_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/search.png</normaloff>
          <activeon>:/icons/icons/search_fav.png</activeon>:/icons/icons/search.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="about_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/about.png</normaloff>
          <activeon>:/icons/icons/about_fav.png</activeon>:/icons/icons/about.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>276</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="hide_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
outline: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/icons/icons/right_arrow.png</normaloff>
          <disabledon>:/icons/icons/right_arrow.png</disabledon>
          <activeon>:/icons/icons/left_arrow.png</activeon>:/icons/icons/right_arrow.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="settings_stack">
     <property name="minimumSize">
      <size>
       <width>244</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>244</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60,60,60)</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="favourites_page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="4" column="0">
        <widget class="PersonSettings" name="person_settings_fav" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="ImportDatabase" name="import_database_fav" native="true"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>424</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="EventSettings" name="event_settings_fav" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="NewDatabase" name="new_database_fav" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="SaveFile" name="save_widget_fav" native="true"/>
       </item>
       <item row="7" column="0">
        <widget class="SearchMailSetting" name="find_mail_fav" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="database_page">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="NewDatabase" name="new_database" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>440</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="ImportDatabase" name="import_database" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="person_page">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="SaveFile" name="save_widget_person" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>450</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="PersonSettings" name="person_settings" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="event_page">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="SaveFile" name="save_widget_event" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>450</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="EventSettings" name="event_settings" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="find_page">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="SearchMailSetting" name="find_mail" native="true"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>444</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="about_page">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0">
        <widget class="QLabel" name="lgpl_logo">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../resources/resources.qrc">:/icons/icons/lgplv3.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="copyright_text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>Copyright Jurij Robba 2016</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p&gt;Ta program je zaščiten z LGPLv 3 licenco, ki omogoča:&lt;/p&gt;
&lt;ul &gt;
&lt;li&gt;Svobodo, da upoorabite program za kakršenkoli namen.&lt;/li&gt;
&lt;li&gt;Svobodo, da spremenite program po svojih potrebah. Originalna koda je na &lt;a href=&quot;https://github.com/&quot;&gt;&lt;span style=&quot; text-decoration: none; color:#008000;&quot;&gt;Githubu&lt;/span&gt;&lt;/a&gt;.&lt;/li&gt;
&lt;li&gt;Svobodo, da delite program s komerkoli hočete.&lt;/li&gt;
&lt;li&gt;Svobodo, da delite modificirane verzije programa.&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;
&lt;p&gt;Edina omejitev je, da ta program in vse verzije vklučujejo in upoštevajo to licenco.&lt;/p&gt;
</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SaveFile</class>
   <extends>QWidget</extends>
   <header>gui/settings/savefile.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewDatabase</class>
   <extends>QWidget</extends>
   <header>gui/settings/newdatabase.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImportDatabase</class>
   <extends>QWidget</extends>
   <header>gui/settings/importdatabase.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PersonSettings</class>
   <extends>QWidget</extends>
   <header>gui/settings/personsettings.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EventSettings</class>
   <extends>QWidget</extends>
   <header>gui/settings/eventsettings.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchMailSetting</class>
   <extends>QWidget</extends>
   <header>gui/settings/searchmailsetting.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
